geowebcache/pom.xml

```diff
@@ -39,7 +39,7 @@
     <joda-time.version>2.8.1</joda-time.version>
     <fmt.action>format</fmt.action>
     <fmt.skip>false</fmt.skip>
-    <jackson.version>2.8.7</jackson.version>
+    <jackson.version>2.9.7</jackson.version>
```

Comment:
I’d like to verify geoserver has been likewise updated for these other dependencies